### DISCUSSION TITLE
suggestion

### DISCUSSION THREAD

**Discussion by ILUVHINATA** [jasonjgardner/jg-rtx#16] [Feedback]
perhaps you can set all texture resolutions to 128x and make the ores glowing and highlighted? let me know if you can/will..
would like to see it, and i am eagerly waiting for the chests and beds and other things like that to be done. :)

**Comment by jasonjgardner** [comment]
Thanks for the suggestion! I've recently added some build tools which will allow me to create some alternate versions of JG RTX, including a 128px version, a pre-baked version for players without RTX, and an unbaked version for use in RTX-enabled worlds and Mineways/Omniverse.

I've created a few textures for chests and beds which will be included in the next release. More chests, beds, and glowing ores coming soon after!

Just curious, why are you interested in a 128px version? Personally, I'd rather make higher resolution textures (512px+) but Minecraft probably couldn't manage a _complete_ resource pack greater than 256px.
